src/main/java/core/mate/academy/service/MachineService.java

```diff
@@ -26,13 +27,13 @@ public interface MachineService {
      * @param machines - list of machines to be filled with value
      * @param value    - any object of machine sub class
      */
-    void fill(List<Object> machines, Object value);
+    void fill(List<? super T> machines, T value);
 
     /**
      * Call the method doWork() from each machine.
      * This method should be able to accept a list of bulldozers as well as list of trucks.
      *
      * @param machines - the list of machines
      */
-    void startWorking(List<Object> machines);
+    void startWorking(List<? extends Machine> machines);
```

Comment:
```suggestion
    void startWorking(List<? extends T> machines);
```
And here, you could also change the value of your parameter. However, I'm not completely sure about this, so it's better to wait for the mentor's review.

Reply:
++

Reply:
not fixed